### ISSUE TITLE
uses_measurement_unit

### ISSUE THREAD

**Issue by romanilin325** [CommonCoreOntology/CommonCoreOntologies#497]
The relation uses_measurement_unit has domain Information Bearing Entity, should not it be Information Content Entity?

**Comment by alanruttenberg** [comment]
yes it should. Expected to be part of the IBA/IBE refactor coming in, I believe, 2.0

**Comment by ahasanzadeh13** [comment]
On a related matter, most of the CCO Data Properties have `Information Bearing Entity` as their domain.
Why should `has text value` or `has URI value` not have `Information Content Entit(ies)` as their domains?

**Comment by neilotte** [comment]
@mark-jensen - labeling for 2.1 @ahasanzadeh13 please create a separate issue labeled for 2.1 with your concern above. Thanks!
